ETL/Pipeline/test_extract.py

```diff
@@ -0,0 +1,106 @@
+"""This file tests whether the extract file works as expected"""
+from unittest.mock import patch
+import pytest
```

Comment:
```suggestion
"""This file tests whether the extract file works as expected"""

from unittest.mock import patch

import pytest
```
Separates python library from third party libraries